Nerfs missionary/wololo staff

Kyep opened the pull request:
This PR nerfs the missionary/wololo staff, by:
- Giving targets a warning when conversion starts ("Your mind seems foggy. For a moment, all you can think about is serving the greater good... the greater good...") rather than simply informing victims they were converted successfully after its too late for them to do anything about it.
- Changing the time it takes to convert, from 4 seconds (faster than vamp enthrall, despite this being a ranged and renewable ability) to 8 seconds (slower than vamp enthrall).
- Making the enthrall ability of the staff never work against mindshielded crew, rather than still work 20% of the time, without even removing the mindshield, as it does currently.

Currently, the staff is horribly overpowered, as it is a hard-to-counter, quick acting, ranged enthrall that can be repeated as many times as you like throughout a round, limited only by its recharge. Worse, it doesn't even notify you that you're being targeted by it until you're already enthralled by it. And you can be targeted with it anywhere the wielder can see, even potentially through doors or walls. These nerfs are aimed at making it more fair (you have a chance to escape it if you're paying attention) and less overpowered.

🆑 Kyep
tweak: Victims of the missionary/wololo staff are now warned when the staff starts being used on them, it takes 8 seconds to convert (rather than 4) and the staff no longer works on mindshielded crew.
/🆑

Ralta commented:
That branch name. Yes.

SamHPurp commented:
The staff needs a nerf, yes.

This however is too much... after a few zealots I'd be breaking bones with every wololo... and before long, straight up gib myself from full hp.

Considering how many convertees just run away when you wololo them and don't listen to you on top of that....

Kyep commented:
To be really blunt, @SamHPurp I'd prefer to remove the staff entirely. I hate it. Its an uncounterable, ranged, nigh-instant conversion that can be used many times, sometimes even works on mindshielded people, and is generally extremely overpowered and frustrating for everyone except the Chaplain.

However, since I don't think a straight-out removal of the staff from the game would pass, I'm nerfing it instead.

The damage dealt here isn't spectacular, and remember, you can simply walk into medical, take a bunch of healing chems (it only does BRUTE) and be good. If you're running around spamming it, to the point you use it 8 times or more... then yeah, I hate to have to say it, but it should kill you at that point. That's just way too much. The wololo staff should be a rarely used thing, not something Chaplains run around spamming all shift.

uraniummeltdown commented:
Recruiting or becoming a thrall is very fun. You don't often see the Chaplain actually have a group of acolytes following him. Sure you could nerf it for mindshielded people, but doubling damage like this sucks.
It wouldn't be so bad if you changed the damage time from Brute to Burn or Tox. Brute is a pain in the ass because of the bone breaking, and broken bones hurt organs adding insult to injury

SamHPurp commented:
> To be really blunt, @SamHPurp I'd prefer to remove the staff entirely. I hate it.

Then do so, nerfing this hard is functionally the same.

Kyep commented:
No. For several reasons:
- Nerfing it like this is not remotely the same as removing it. With this PR, all you need to do is grab some healing stuff from medbay, or even use your own bible, and its good for quite a few uses. Heck, you can get a few without even that. The upper limit is in place to stop you abusing the heck out of it with no consequences. NOT to effectively remove it.
- Outright removal PRs almost never pass, so I don't fancy taking on those odds.
- PRs aren't supposed to be solely based on the author's personal preferences. While I might personally strongly dislike the staff and think we'd be better off without it, or at least, with it being subject to major changes, I am not certain everyone else agrees. Most people I've seen bring it up consider the staff overpowered, or just memey, but aren't calling for its outright removal. They just want it to be less overpowered. Hence, a nerf.

TullyBurnalot commented:
Myself and the Maintainers were discussing this, and we'd like to propose an alternate nerf:

A) Staff no longer works on Mindshielded personnel
B) Add a "casting time" to the staff where both individuals have to remain still
C) Optionally, limit the number of maximum thralls.

@Kyep

FalseIncarnate commented:
@Kyep Please don't mislead users.

The staff is not instant. It takes 4 seconds to convert someone, during which time the missionary must remain still and able to see their target (moving, dropping staff, or the target leaving your sight will all break the attempt)

Mindshielded crew have an increased chance for the conversion to fail, and make the cooldown between conversions longer than normal when they DO get converted. Additionally, mindshielded crew only get converted for half the duration of their non-implanted crewmates.

Failed attempts also incur a loss of faith, which requires time to regenerate (except in cases like attempting to target a mindless mob or yourself), so this is hardly spammable.

If you're going to nerf it, increase the recharge time (reduce the faith recharged per cycle) or decrease the conversion chance on mindshielded crew. Making it deal damage to the user doesn't fit the item's origins, and doesn't really affect what you believe to be the underlying issue. It just punishes someone for being able to succeed.

SamHPurp commented:
@TullyBurnalot 

Agree with C) as long as on C) you mean concurrent, rather than grand total.

I don't necessarily think A) needs to be done. The strength of the staff isn't really made much weaker by the loss of that, plus as @FalseIncarnate says there are all sorts of weaknesses in attempting to use this on mindshielded staff already. Plus, the best person to wololo is the HoP to give you all access etc anyway... who isn't mindshielded.

B) would be a /significant/ nerf which is probably an appropriate one, but it is stronger than you might initially think.

A less impactful version of B could be: You must maintain a clear line of sight on them. As one tactic at the moment is to wololo through walls and other things.

Rurik123 commented:
This seems like a odd way to go about nerfing the staff. As SamHPurp has mentioned, this too much of a nerf, and as FalseIncarnate mentioned it doesn't affect the underlying issue. I feel both A and B options of what Tully stated would suit this far better.

Citinited commented:
Flavour-wise, how does a missionary staff dealing brute damage to its owner make sense? You're not doing anything physical with it, and the effect is entirely on another crewmember, yet it still breaks your leg?

SamHPurp commented:
@Citinited nah I totally get the flavour stuff, just it could be done better.... there was/is a lot of self-flagellation...

Citinited commented:
@SamHPurp Makes sense, actually. Still needs text for that though 🤔

Kyep commented:
@TullyBurnalot 
- So, I'm glad the maints/heads were discussing it, but I'm not convinced those nerfs are enough, because (A) it already has an 80% chance to fail on mindshielded personnel. The success rate on mindshielded personnel isn't an issue, the issue with mindshields is that it can work AT ALL and the person doesn't lose their mindshield, so there is no way for everyone else on the sec team to know it worked. (B) it may have a few seconds of casting time already, but compared to say, vampire enthrall, this is nothing, especially since you don't need to be next to them. Hence the 'almost instant' - compared to having to abduct someone to enthrall them (which risks being caught) (C) do you mean limiting the maximum number of thralls in total, or the maximum number of thralls at a time? If it is the former, I agree. If it is the latter, I think that's too soft, as it isn't usually concurrent thralls that are the problem - usually it is just spamming it throughout the shift that's the problem.

@Falseincarnate
- I still consider 4 seconds "almost instant", as I said above, versus other types of enthralling methods, like vamp thrall, which essentially require you to abduct the person, and can take a minute or more. Expecting people to react within 4 seconds when they're standing still, doing something, is unrealistic.
- I understand how mindshielded conversions work. My point is that most types of enthralling won't work AT ALL on mindshielded personnel, and the ones strong enough to work, at least remove the mindshield so other people can tell it worked. Neither of those is the case here.
- The issue isn't that the staff is spammable within, say, a minute. The issue is that you can keep using it throughout the shift, it never completely runs out, and there is no consequence for using it continually.
- I am not convinced that increasing the recharge time will help. BUT you are right that "user takes damage" is a bit of a lazy way to nerf it.
- I like the idea of nerfing its effects on mindshielded crew. Frankly, I'm inlinced to have it simply never work against mindshielded people.

Kyep commented:
**Major change: the damage nerf has been removed from this PR. Instead, I've changed the duration it takes to enthrall someone from 4 seconds to 8, and made it never work against mindshielded folks.

IE: Tully's options A & B.**

Kyep commented:
# Nerfs changed!
- Removed all damage-on-convert.
- Added warning when you START to convert someone, so that they at least get a warning about what is happening. Without this, how would they even know they need to move?
- Changed duration of convert from 4s (faster than vamp enthrall, despite being a renewable ability, and being ranged) to 8s (slower than vamp enthrall).
- Prevented it working at all on mindshielded personnel.

SamHPurp commented:
8 seconds long and a warning when you start? lol. I guess you'll only use this on someone you've already stunned and cuffed then. The warning is a ridiculous concept.

Kyep commented:
God forbid people actually have a chance to fight back against / resist the wololo staff.
As they do with things like mindshield implants, vamp enthralls, etc.
Perish the thought that Chaplains using the staff might actually have to abduct people, or at least time their enthralls, rather than simply wandering using the staff on people until it works, safe in the knowledge that failed attempts will never give them away.
Oh, isn't it so terrible that people who are being targeted with a brainwashing ability might, y'know, notice?

SamHPurp commented:
You want to turn a **hijack only** item into something thoroughly shit and against the entire purpose by forcing people to "abduct then convert".

I disagree with this even more than your silly brute damage idea.

Kyep commented:
No. I'm fine with chaplains converting people who aren't paying attention without abducting. If they have to abduct because people aren't going to let them effectively walk up and inject them with a mindshield, good! But, equally, if they aren't paying attention and let people do it, fine, that's equivalent to mindslave implants, which is OK.

What I am not fine with is the staff being effectively uncounterable, because people don't get a warning when they're targeted with it. What angers me (and others) about the staff is that the first hint its being used is when you're already thralled by it. There's no counterplay to it.

SamHPurp commented:
'Not paying attention for 8 seconds' - you mean AFK or in another tab. Those are the only people you'll convert. You'll only convert the exact same people that you could have walked up to and wordlessly mindslave implanted. There is literally zero difference. Except the mindslave implant is actually faster.

This is one of the few items, particularly **for hijackers only** that result in a net gain in fun for players, as more people are now 'in on the evil-doings'. 

I condemn your proposed changes in the strongest possible terms.

FalseIncarnate commented:
@SamHPurp Missionary kit is not hijack only.

There doesn't need to be a warning to this though, this is a somewhat risky choice because of the fact that both you and the victim end up shouting "WOLOLO!" whenever you convert someone. If you're doing this in public anyways, you're not gonna get too far without being valided.

Combine that with the fact that if you are standing around near a stationary chaplain holding a big ass staff for more than a few seconds, it's either by choice, against your will anyways, or you really aren't paying enough attention to see a warning message anyways.

KingPhilipIII commented:
I've had the staff twice as a nonchaplain, curtesy of a surplus crate, and I just used it to meme by starting a cult in the chapel. When an officer started sniffing for valids I wololo'd her and told her to shove off. Came back after it wore off and tased me and dragged me off.

I mean. My acolytes mobbed and beat her to death but that's besides the point. I don't think this is nearly enough of an issue to justify such a major nerf, rendering the kit almost useless, not to mention removing one of the few ways in a traitor round to actually get people in on the evil.  Not to mention this is already a rarely used item because you're gambling your round on not running into valid hunters, and while this is already the case for some items, atleast those items let you properly defend yourself. 

I'm not entirely opposed to a nerf, but in it's current form I do not support this PR.

FalseIncarnate commented:
@KingPhilipIII that is a hilarious story, though I do want to point out the sec officer is specifically told when the conversion wears off that they do not remember anything regarding the conversion. Unless they had reports of your activities or such, they should not have come back for you when it wore off.

KingPhilipIII commented:
Nobody had reported me to my knowledge. I was keeping to myself in the chapel and mostly just fucking around telling people to retrieve holy artifacts from various departments, nothing major though, or conducting witch hunts by swirly. I'd even done it in front of a squad of officers that came through pursuing someone but I was actually roleplaying and preaching to my flock so they didn't bother me at the time.

Ultimately I suppose she got what she deserved for seeking them valids, but I generally use the staff to fuck around as opposed to actual malicious things.

Kyep commented:
My issue with the staff is that, right now, its effectively uncounterable for most crew.
You're simply standing there, minding your business, and suddenly, WOLOLO brainwash-memes, before you even realized a Chaplain was around, let alone you were being attacked.

This sort of surprise bullshit is not fun or balanced.

The point of this PR is to rebalance it so that:
A) People have a realistic chance of actually resisting it if they're paying attention and have not been abducted.
B) It isn't so ridiculously OP.

Put very simply, you HAVE TO warn people if they're being wololo'ed, BEFORE they've been converted. It is impossible to effectively resist the staff otherwise.
What are you gonna do, run every time you see a Chaplain with a staff? Even the reskinned null rods that (IIRC) have identical sprites? Not practical.

Four seconds isn't even enough time to notice the warning text in your chat box, realize it applies to you, and read it. 8 seconds is much more reasonable.
Give them 8 seconds. If they're not paying attention, kidnapped, or whatever, you've still got them. But at least they have a chance to resist it, just like vamp enthrall, shadowlings, etc. They have some warning about what is happening and can't just get risk-free thralled in public corridors with no realistic way to know it was happening until after it is done.

Thralling generally doesn't work on mindshielded people, at least not without removing the mindshield. Currently, this not only has a chance to work, it doesn't even remove the mindshield so there's no way to tell someone in the sec team is thralled by it. That's doubly bad. So, no more working on mindshielded people, even at a 20% chance. 

I like engaging play. I even like allowing other people to get in on the antag action. I just don't think it should be as easy, uncounterable and risk-free as it is currently. I absolutely hate it when I'm playing a character and I get wololo'ed by a passing Chaplain, for memes, with no opportunity to even know it was happening until after I'm converted. At the very least, people need a fair chance to resist, and that means they have to (A) know when it starts happening (B) have a few seconds to be able to notice the warning and move/run, and (C) be able to take preventative measures, like mindshields. With those three conditions met, its fair. Still very strong, compared to say mindslave implants which are more obvious AND melee-only. But at least it isn't totally overpowered like it is now.

SamHPurp commented:
My proposed fixes:

- [ ] Mark it hijack only.

- [ ] Remove it from the Surplus Crate

- [ ] Keep the four second timer, but if they move at all - rather than out of range, it cancels.

- [ ] Ensure line of sight (no using x-ray and doing it through walls etc.)

Kyep commented:
Hijack only would make sense. Not sure about surplus crate. Four seconds isn't long enough to react, though, and it needs some notification still or there's zero chance you can realize its being used on you before its too late. Line of sight is tricky. I suppose you could make it require that the target can see you.... but that would imply odd things, such as blind people never being able to be targets, not sure if it would work in the dark, etc.

Purpose, how about I just make it 8 seconds, with notification, and hijack only? :P (Kidding, not going to do this unless the heads ask for it)

Vivalas commented:
I would rather either a warning or text, or maybe both, but I feel like this is a fun police nerf. Please don't destroy one of the few creative traitor items. This is effectively a removal because with the nerfs in addition to large cost, it will fall down to the depths of uplink hell. I think it is appropriately balanced for such a costly item, because not everything needs to be perfectly balanced.

Dumbdumn5 commented:
The problem I always had with this item was that you could effectively stand across a room and, given somebody was alone in a room at all, you could pretty much convert them with very little chance of them disarming you or running away due to the four second countdown. A warning and cancellation on movement or a warning and a greater time limit, only for the convert, would make a bit more sense because otherwise the chaplain could pretty much just chill out in the corner of a room with the staff and you'd have absolutely no clue he was converting you. 

I don't think this is as far as nuking it off the face of the earth, as it's more or less an infinite ten minute mind slave machine with a recharge rate of about 2 minutes if all attempts are successful. You could effectively lock in a good five people to your service if you had the timing down with very little time between realizing their deconversion and simply being converted again.

The goal of an antag is to make the round more fun, yes, but this item seems like something that's much more fun to fight with than fight against, due to its high range, low channeling time with no warning, multiple thralls, cult deconversion style forgetfulness, and a cooldown, if I recall the numbers correctly, of about a fifth of successful conversion.

I'd take a look at either the warning with eight seconds or maybe a warning on 4-6 seconds cancelling if the other party moves, as, without the warning, you could pretty much just wear dark clothes, plop yourself down behind a potted plant sprite, and convert from afar with pretty much no worries of ever being found out by a potential victim before it's too late.

While not everything has to be balanced, it does help to make everything at least have a fair chance of being fought back against if you've got any clue what you're doing. The current four second time limit with no warning that can be done so long as the chaplain can see you is a little bit more than reasonable, as every other conversion method requires being next to a player, both players remaining still for a lengthy period of time, warnings on every single one of them, and a series of events that make it obvious that you'll lose player agency if this thing touches you.

adamkad1 commented:
well that certainly makes the staff nearly useless. gonna have to go back to murdering people for my goals, and its all your fault their characters are permanently removed from the round. mindslaving wardens and getting their gloves peacefully was so much fun... I never converted more than one person at time since i got better ways to do hijack and murder than making myself a ton of mindslaves, and getting a item usually requires only one person

SamHPurp commented:
@Kyep And in my mind it CANNOT have a notification to the target. Its pretty obvious with an Chaplain standing staring at you with a big ass staff.

It takes one person to mention the wolololo , or shout about the conversion attempt and that guys round is over.

Adding any notification at all removes the viability of use of the item, and as @alexkar598 states, you end up just encouraging your standard murder techniques rather than an item that includes users in the dirty deeds, and adds at least a little bit to the roleplay. (Huh, whats going on, why am I here? Why am I standing over the HoS's bloody corpse... WHAT HAVE I DONE!)

ktoma36 commented:
Considering the cost of this item and how easy it is to get screwed out of a round with it I disagree with this nerf almost entirely. Why make every conversion action the same? Why not come up with more creative ways to make this item less powerful while still making it worth pouring 15 Tc's into it.

Here are a couple ideas I have for the staff:
- Have the orb glow red for a few seconds while channeling
- Make the caster whisper a prayer, something like, *"By my faith you shal see the light"* while casting
- Give the mindslaved person a little noise or something alongside the alert that they have been converted as well as clear up the text blurb that they get sent (Cause I really bloody hate it when I convert someone, give them a command, and they tell my to bugger off/ start attacking me)
- Give the person with the staff long white hair (Or have the robes come with a wig)
- Give the robe a unique sprite and name (It looks nothing like the robe that the priest wear in Age of Empires)
- Give the mindslaved individual a unique flavor text indication that they are under mind control (though this would quite clearly throw suspicion on the chaplain as soon as someone sees it)
- People who have mindshields that get converted still keep their implant, though on huds their indicator would be blinking/ some other effect to signify that they are converted. Regular crew without mindshields would not give this indicator (just in case someone misreads the first part).

These are ideas off the top of my head so if they seem silly they might very well be. The item is meant to be silly and at the cost of 15 TC it should be powerful.

And as for the complaint that it isn't counter-able therefor bad there are many other items that cannot be countered *and* can be used indefinitely. For example a sleepy pen with Perfluorodecalin and Capulettium. Only 8 TC and stealthy. How about a smoke bomb with black slime potion? No TC and can wipe an entire room. There are much cheaper and reliable ways to go about traitoring, the wololo staff is a high risk high reward item. Perhaps bumping the time up for conversion to 6 seconds would be good, but notifying the person beforehand would make the staff useless.

adamkad1 commented:
i agree with @ktoma36 those sound like actually good balance changes

alexkar598 commented:
@SamHPurp I stated nothing I think you clicked enter too fast while typing adamkad1 's name

Anoonki commented:
While you're at it, assuming it hasn't been fixed.
Could you address the fact that this can be used through walls by people who can't see you, but have thermals? Or is that intended?

ktoma36 commented:
Perhaps give it an additional chance to fail if used through a wall, with the caster shouting out loud the prayer on a successful enthrall. Or make it take more faith with a longer recharge. Or both!

Fox-McCloud commented on code/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -519,6 +519,8 @@
 	var/obj/item/clothing/suit/hooded/chaplain_hoodie/missionary_robe/robes = null		//the robes linked with this staff
 	var/faith = 99	//a conversion requires 100 faith to attempt. faith recharges over time while you are wearing missionary robes that have been linked to the staff.
 
+	var/zealots_made = 0
```
No point to this now.

Kyep replied:
Removed.

KingPhilipIII commented:
The person doing the conversion already shouts 'WOLOLO!' on a successful conversion.

SamHPurp commented:
@KingPhilipIII I believe even the convertee shouts it too, giving further range on the people that could hear it.

FalseIncarnate commented:
@ktoma36 the missionary is the only one who shouts WOLOLO, but both the missionary and the convert have the WOLOLO sound clip played to them upon conversion.

The robe doesn't look like the one from AoE because it would be an outright giveaway. It instead looked exactly like the robes a chaplain normally has to not be a giant "valid me" sign. This goes for the names of the items as well. Anything named differently than what is normal is immediately going to be used against the user (see the contortionist's jumpsuit which is named differently than the atmospheric technician jumpsuit it is meant to appear as; an examine immediately gives away that its not the normal suit they should be wearing)

Anything that outright tells people the convert is a zealot (flavor text, flashing HUD icons, etc) is a horrible idea for the same reasoning. People WILL meta-knowledge that and immediately valid (or as close to it as they can) the zealot for having been victimized in the first place.

If people are betraying you when you convert them, you need to ahelp that since they are violating their new role just as if they were to do the same upon being culted, thralled by a shadowling, or so on. They are given a sound and a message, and if they ignore both that is entirely on them.

FalseIncarnate commented:
The ability to convert someone you can see but that cannot see you is intended as well. Yes, that means you could hide in a locker, or make use of things like xray vision.

The fog of war is not mutual, you only can see what is within your own sight range, but that doesn't mean they can't see you from beyond it.

tigercat2000 commented:
After discussion between the Heads of Staff, Maintainers and original PR author, this PR is being merged; the staff in its original incarnation offered a rechargeable, effectively endless mindslave on a timer with absolutely no counters whatsoever, merely "Click and go". Therefore, it is being reduced in its effectiveness to address balance concerns.

Doukhan commented:
Yet another utterly useless item then.

Some vague hint or even an absolutely obvious indicator pre/post immediate conversation would have been sensible but seeing as the approach to everything is to meta the shit out of it RIP the staff _ever being usable nevermind practical._

ktoma36 commented:
... kept **both** the 8 second timer and the warning message.... Down to the bottom of uplink hell to this traitor item

DesolateG commented:
Might as well have removed it.

The-Driker commented:
On a potentially no longer relevant note. I've stood directly next to Chaplin's as they wololo people and haven't heard the wololo shout. The only time I've ever heard it, was when I myself was wololo'd. I suspect that only the Chaplin and the victim see/hear the wololo after the conversation is successful.

SamHPurp commented:
Its like nerfing a car by locking the axles.

ktoma36 commented:
Instead of trying to make this item unique with some of the suggestions we have been putting in all that was done was make it basically make it scream "The chaplain is a traitor!" to anyone the chaplain attempts to convert. Anyone who uses this now runs the risk of the victim shouting "HELP WOLOLO" over comms after moving out of LOS, then walking back into LOS to beat the crap out of the chaplain. In 8 seconds a person could easily critically harm or even kill the chaplain in question. With the price of the staff still being 15 TC you cant even buy an esword to fend off the inevitable lynch mob that comes after you. In favor of making this nerf come out ASAP Kyep ignored suggestions made by numerous people, some whom have made numerous post in the past and are highly respected.

To put this simply this nerf was put together fast, the changes were not tested, and the reasoning for the changes did not make sense `

Crew will notice if a vamp tries to enthrall them. If someone tries to inject them with a mindslave. If a shadowling starts putting hands on their head.There is NO reason that people shouldn't be given a warning about this type of mind-control, too.You cannot expect people to have any fair hope of resisting a type of mind control if they're never even notified it is happening until its too late. That's not fair or balanced, especially since, unlike shadowling/vamp/etc Chaplains can and do regularly use this staff in main hallwayas, and get away with it.Now, Chaplains will have to at least time their brainwashing, if not play by the same "abduct then convert" rules that other conversion antags play by. This will go a long way towards making the missionary staff fair and counterable, as opposed to being ridiculously overpowered as it is now.
--
`
`

I don't like that proposal. As far as I'm concerned, it HAS TO notify the target that its trying to mindslave them, and give them enough time to react. Otherwise, it is unfair and unfun, because you can't counter it.
--

`
`My issue with the staff is that, right now, its effectively uncounterable for most crew.
You're simply standing there, minding your business, and suddenly, WOLOLO brainwash-memes, before you even realized a Chaplain was around, let alone you were being attacked.

This sort of surprise bullshit is not fun or balanced.

The point of this PR is to rebalance it so that:
A) People have a realistic chance of actually resisting it if they're paying attention and have not been abducted.
B) It isn't so ridiculously OP.

Put very simply, you HAVE TO warn people if they're being wololo'ed, BEFORE they've been converted. It is impossible to effectively resist the staff otherwise.
What are you gonna do, run every time you see a Chaplain with a staff? Even the reskinned null rods that (IIRC) have identical sprites? Not practical.

Four seconds isn't even enough time to notice the warning text in your chat box, realize it applies to you, and read it. 8 seconds is much more reasonable.
Give them 8 seconds. If they're not paying attention, kidnapped, or whatever, you've still got them. But at least they have a chance to resist it, just like vamp enthrall, shadowlings, etc. They have some warning about what is happening and can't just get risk-free thralled in public corridors with no realistic way to know it was happening until after it is done.

Thralling generally doesn't work on mindshielded people, at least not without removing the mindshield. Currently, this not only has a chance to work, it doesn't even remove the mindshield so there's no way to tell someone in the sec team is thralled by it. That's doubly bad. So, no more working on mindshielded people, even at a 20% chance.

I like engaging play. I even like allowing other people to get in on the antag action. I just don't think it should be as easy, uncounterable and risk-free as it is currently. I absolutely hate it when I'm playing a character and I get wololo'ed by a passing Chaplain, for memes, with no opportunity to even know it was happening until after I'm converted. At the very least, people need a fair chance to resist, and that means they have to (A) know when it starts happening (B) have a few seconds to be able to notice the warning and move/run, and (C) be able to take preventative measures, like mindshields. With those three conditions met, its fair. Still very strong, compared to say mindslave implants which are more obvious AND melee-only. But at least it isn't totally overpowered like it is now.`


Each time Kyep just repeated what he said before without addressing the concerns or suggestions made by others. The logic is flawed: `This sort of surprise bullshit is not fun or balanced.

The point of this PR is to rebalance it so that:
A) People have a realistic chance of actually resisting it if they're paying attention and have not been abducted.
B) It isn't so ridiculously OP.` 
`I don't like that proposal. As far as I'm concerned, it HAS TO notify the target that its trying to mindslave them, and give them enough time to react. Otherwise, it is unfair and unfun, because you can't counter it.`
Unfair and unfun are completely opinionated terms, and there are many much more viable options for a traitor. Especially considering the staff _still cost 15 TC_.

`At the very least, people need a fair chance to resist, and that means they have to (A) know when it starts happening (B) have a few seconds to be able to notice the warning and move/run, and (C) be able to take preventative measures, like mindshields. With those three conditions met, its fair. Still very strong, compared to say mindslave implants which are more obvious AND melee-only. But at least it isn't totally overpowered like it is now.``
The staff might as well be melee only if the only viable way to use it is to kidnap, strip, then convert. Range means nothing when the person is given ample time to run or disarm you.
The biggest flaw in the argument that Kyep never addressed was the fact that if anyone exposes the chaplain their round is over. You cant defend yourself with that big magic staff, cant hide it in a bag, cant even wear a hardsuit to hide your identity. You spent 15 TC on what has effectively become a robe with "Valid" stitched on the back. And if your staff gets stolen you are pretty much SOL for the round.

This PR comes off as hostile towards the community, especially sense the reasoning stated by Kyep seems to have come from personal experience. When given suggestions by respected members of the community they were ignored, and in some cases seemingly attacked. The worst part is that I know by experience any PR's trying to actually use some of the suggestions made by the community will be shot down under the reasoning "There hasnt been enough time to properly judge the effects of the nerf". This PR is doing exactly what Kyep wanted in the first place. That is removing it from the game, only instead of deleting it the staff is just being nerfed to uselessness. 
`Outright removal PRs almost never pass, so I don't fancy taking on those odds.`

Vivalas commented:
Less go paradise, keep merging shit the community disapproves of....

Kyep commented:
@Vivalas 
This is not intended to be a fun police nerf. Even if it was, it wouldn't matter, because a chaplain wololoing 6 people in a round might be having lots of fun, but those 6 people who got wololoed with no idea it was even happening until after they're converted... those peoples' fun matters too. Conversion stuff that can hit you in main corridors and gives you no knowledge that it is happening, or chance to counter, is not fun.

@Dumbdumn5 
You're right.

@adamkad1
IMHO, you shouldn't be able to mindslave the Warden, a mindshielded member of security with armory access, and often the acting HoS, so easily. IMHO, the fact you could was blatently OP. 

@SamHPurp @Doukhan
The notification is to the person being targeted, NOT everyone in the vicinity. All your worries about everyone else in the area instantly validing the chaplain for clicking with this are unfounded. Most of the time, they'll still target distracted people (like someone using a console) who probably won't notice before they're thralled. This doesn't make it guaranteed they'll fight back, this just gives them a chance.

@ktoma36
I did not make the sprite glow red as I am not a spriter and try to avoid sprite-related PRs as a result.
I did not make the caster whisper as that's inadequate.
I did not change the on-convert text the way you describe as that wasn't the point of this PR. Feel free to make your own PR doing so.
I did not make the caster use unique robe sprites, white hair, etc, again as I'm not a spriter.
I did not give zealots unique flavor text as that would make the item useless.
I did not change things for mindshielded crew as you describe because the entire purpose of mindshields is to block mind control, I don't think its reasonable for them to have a failure chance in a situation like this, considering they stand up to more powerful conversion attempts by either working or burning out entirely.
Comparing this (an infinitely-rechargable weapon) to a sleepy pen (two uses), or a smoke bomb (has to be made, one use) is not accurate.

@ktoma36
I worked with the heads/maints to figure out what was mergable and what wasn't. While I do appreciate the feedback/ideas, bear in mind that frankly, many of them were totally impossible for various reasons, such as me not having the necessary spriting skill, me being completely opposed to them for balance reasons, the heads/maints not being willing to merge them, etc. I did read and consider everything, but this was the best compromise in terms of what I felt addressed the issues, without removing the staff entirely, and was mergable. I did not ignore any suggestions, yours or otherwise. I read and thought about everything posted. This was still the best compromise that could be reached, especially bearing in mind various people involved had certain incompatible fixed opinions that were not going to change anytime soon. If you really hate this PR, you're more than welcome to put yourself in my shoes and try to work out a fix / compromise that everyone is happy with.

Doukhan commented:
"THE CHAP HAS A MINDSLAVE STAFF HE'S TRYING TO CONVERT ME GET HIM"  cue lynch mob which will be let slide because valid lol.

That will happen _every time_ barring either ignorance or good sportsmanship on the part of the would be convert, both of which are in very short supply. You how people badly people freak out over sleepy pens? Same shit here.

ktoma36 commented:
@Kyep  You can refill the sleepy pin with any chem you like albeit slowly via dropper. I know it is impossible to make everyone happy, its just that the majority of the people who were actually contributing to the discussions were not happy with the end result. Hopefully in the next month or so someone will come up with a PR to revamp the staff

Doukhan commented:
Hate to necro, but it has to be said. How objectively unusable is the staff now? I just found out, took all of 5 seconds for me to be disarm spammed when I tried it.

It is completely and utterly worthless just as predicted, this should never have been merged.

KingPhilipIII commented:
I hate to necro as well but for further emphasis I saw someone with a wololo staff a couple rounds ago. Couldn't get a single convert, was attacked by attempted converts, and after a few minutes of the crew screaming mind control stick(While some of the crew that aren't dicks brushed it off as silly) security arrested them for mind control. 

This item really is the perfect way to ruin your round now, for the low price of 15 crystals. You too can have a useless item that immediately gets you attacked and arrested.

ktoma36 commented:
@Kyep Hey, you know the idea I had about making the staff change colors. Isnt there already two identical sprites for staff? I was just thinking about it and thought that could be a good idea. Sorry to necro as well too

FalseIncarnate commented:
@ktoma36 the two sprites are for the "team" colors (red and blue), not for some sort of cast indicator.
